lake_hylia.logic

```diff
@@ -1,3 +1,90 @@
 area LakeHylia:
-  room BeanstalkArea
-  room Main
+  room BeanstalkArea:
+    node EastArea:
+      door LeftLadder
+      door RightLadder
+    node WestArea:
+      door Ladder
+    node BeanstalkFusion:
+      door BeanstalkFusion
+    WestArea <-> BeanstalkFusion: flag HyliaBeanstalkFusion
+    node EastArea +: exit Virtual.Zeffa.FluteAway
+    node WestArea +: exit Virtual.Zeffa.FluteAway
+  room Main:
+    node Crest
+      entrance Crest
+      door MinishHouse
+      exit CrestHoleFusion
+      flag Virtual.Zeffa.LakeCrest
+    Crest -> NorthWest
+    node CrestHoleFusion:
+      entrance CrestHoleFusion
+    Crest -> CrestHoleFusion: flag CrestHoleFusion
+    node MinishWater:
+      exit MinishSouth
+      exit NorthHole
+    node MinishWaterEntrances:
+      entrance South
+      entrance NorthHole
+    MinishWater -> MinishWaterEntrances: item Flippers
+    node EastVineHole:
+      door EastVineHole
+    MinishWater <-> EastVineHole: item Flippers
+    node MinishPortal:
+      door MinishPath
+    MinishPortal <-> MinishWater: item Flippers
+    MinishPortal -> SouthEast: item CanGrow
+    node MinishPortalBonk:
+      flag HyliaTreeBonk
```

Comment:
No need to include Hylia in the flag name (since it's `LakeHylia.Main.HyliaTreeBonk` that way)